### PR TITLE
change auth-clients docs

### PR DESCRIPTION
**Pull request by VitaliyKovalchuk**

### CODE REVIEW

File: source/includes/_auth-clients.md [hunk 1]
```diff
@@ -92,9 +92,9 @@ This request is authorized with one of the `tenants.auth_clients.get`, `contract
 | Parameter         | Required              | Description                                                                                          |
 | :---              | :---                  | :---                                                                                                 |
 | filter[component] | no                    | Filter the Auth Clients only for specific component. Must be `id` of `Component`                     |
-| workspace_id      | yes<sup>*</sup>       | Show Auth Clients available in the given workpspace (including contract's and tenant's auth-clients).|
-| contract_id       | yes<sup>*</sup>       | Show Auth Clients available in the given contract (including tenant's auth-clients).                 |
-| tenant_id         | yes<sup>*</sup>       | Show Auth Clients available in the given tenant.                                                     |
+| workspace_id      | no<sup>*</sup>       | Show Auth Clients available in the given workpspace (including contract's and tenant's auth-clients).|
```
Comment (HannaTrotsenko) [inline source/includes/_auth-clients.md:95]: (including contract's, tenant's and global auth-clients).

File: source/includes/_auth-clients.md [hunk 2]
```diff
@@ -92,9 +92,9 @@ This request is authorized with one of the `tenants.auth_clients.get`, `contract
...
+| contract_id       | no<sup>*</sup>       | Show Auth Clients available in the given contract (including tenant's auth-clients).                 |
```
Comment (HannaTrotsenko) [inline source/includes/_auth-clients.md:96]: (including global and tenant's auth-clients).

File: source/includes/_auth-clients.md [hunk 3]
```diff
@@ -92,9 +92,9 @@ This request is authorized with one of the `tenants.auth_clients.get`, `contract
...
+| tenant_id         | no<sup>*</sup>       | Show Auth Clients available in the given tenant.                                                     |
```
Comment (HannaTrotsenko) [inline source/includes/_auth-clients.md:97]: Show Auth Clients available in the given tenant including the global scope.

File: source/includes/_auth-clients.md [hunk 4]
```diff
@@ -92,9 +92,9 @@ This request is authorized with one of the `tenants.auth_clients.get`, `contract
 | Parameter         | Required              | Description                                                                                          |
 | :---              | :---                  | :---                                                                                                 |
 | filter[component] | no                    | Filter the Auth Clients only for specific component. Must be `id` of `Component`                     |
```
Comment (HannaTrotsenko) [inline source/includes/_auth-clients.md:94]: Add `?global=true` as the query parameter

Comment (VitaliyKovalchuk) [reply source/includes/_auth-clients.md:94]: no

**Review by HannaTrotsenko — APPROVED** [review]

File: source/includes/_auth-clients.md [hunk 5]
```diff
@@ -325,9 +323,9 @@ To specify scope of request one of `workspace_id`, `contract_id` or `tenant_id`
 
 | Parameter         | Required  | Description                                                                                          |
 | :---              | :---      | :---                                                                                                 |
-| workspace_id      | no        | Show auth-client available in the given workspace (including tenant's and contract's auth-clients). |
-| contract_id       | no        | Show auth-client available in the given contract (including tenant's auth-clients).                  |
-| tenant_id         | no        | Show auth-client available in the given tenant.                  |
+| workspace_id      | no        | Show auth-client available in the given workspace (including global, tenant's and contract's auth-clients). |
+| contract_id       | no        | Show auth-client available in the given contract (including global and tenant's auth-clients).                  |
+| tenant_id         | no        | Show auth-client available in the given tenant or in global scope.                  |
```
Comment (luzhok) [inline source/includes/_auth-clients.md:328]: why `or ` ? I think here `AND` has to be

**Review by luzhok — APPROVED** [review]

**Pull request merged**